src/VisualStudio/Core/Def/TaskList/VisualStudioTaskListService.cs

```diff
@@ -65,13 +66,13 @@ internal class VisualStudioTaskListService :
                 threadingContext.DisposalToken);
         }
 
-        void IEventListener<object>.StartListening(Workspace workspace, object _)
+        public void Start(VisualStudioWorkspace workspace)
         {
-            if (workspace is VisualStudioWorkspace)
-                _ = StartAsync(workspace);
+            // Fire and forget.
+            _ = StartAsync(workspace);
```

Comment:
❗ Asynchronous work must be tracked by the caller, either by awaiting it (which we don't want to do here) or by using async operation tracking.

Reply:
i tried that here, but it doesn't work as this is fire-and-forget-and-PAUSE.  the last bit being problematic for integration tests that continue waiting until there are no outstanding BG tasks.

Reply:
Wouldn't that wait though only be for things waiting on the "task list" activity or something?